common/rdr/FileInStream.cxx

```diff
@@ -56,7 +56,7 @@ int FileInStream::pos()
   return ftell(file) + ptr - b;
 }
 
-int FileInStream::overrun(int itemSize, int nItems, bool wait)
+int FileInStream::overrun(int itemSize, int nItems, bool /* wait */)
```

Comment:
As you've noticed there will just be lots and lots of these, and they will continue to pop up. Can you handle this via a compiler flag instead?